pdc_client/plugins/component.py

```diff
@@ -64,8 +64,15 @@ def list_global_components(self, args):
             self.subparsers.choices.get('list').error('At least some filter must be used.')
         global_components = get_paged(self.client['global-components']._, **filters)
 
+        # TODO(xchu): this is temporary hack, should be removed after we drop the old contacts
+        global_components = list(global_components)
+        for global_component in global_components:
+            component_contacts = get_paged(self.client['global-component-contacts']._,
+                                           component=global_component['name'])
+            update_component_contacts(global_component, component_contacts)
+
         if args.json:
-            print json.dumps(list(global_components))
+            print json.dumps(global_components)
             return
 
         if global_components:
```

Comment:
If not args.json, the contact information won't be used at all, but above already got them. If global_components already is a list, 'if global_components" looks unnecessary.